main.py

```diff
@@ -8,27 +8,29 @@
 
 try:
     import wandb
+
     has_wandb = True
 except ModuleNotFoundError:
     has_wandb = False
 
 try:
     import safetensors
+
     has_safetensors = True
 except ModuleNotFoundError:
     has_safetensors = False
 
 
 def get_average_number_of_bits(
-    wbits: int = 3,
-    qq_scale_bits: int = 3,
-    qq_zero_bits: int = 3,
-    qqq_scale_bits: int = 16,
-    qqq_zero_bits: int = 16,
-    groupsize: int = 16,
-    qq_groupsize: int = 16,
-    round_zero: bool = False,
-    global_ol_n_share: float = 0.00,
+        wbits: int = 3,
+        qq_scale_bits: int = 3,
+        qq_zero_bits: int = 3,
+        qqq_scale_bits: int = 16,
+        qqq_zero_bits: int = 16,
+        groupsize: int = 16,
+        qq_groupsize: int = 16,
+        round_zero: bool = False,
+        global_ol_n_share: float = 0.00,
 ):
```

Comment:
why excessive indent?

Reply:
Sorry about that, I did not notice it. The reason was Pycharm's, automatic formatting. I'm not sure the changes are easily reversible. If it is critical I could try to revert the formatting, is it critical?

Reply:
As for two arguments instead of one -  in my opinion, in this way it is much clearer what argument doing what.

Reply:
1 argument  -- save it is